src/Sulu/Bundle/SecurityBundle/Resources/public/js/components/roles/edit/main.js

```diff
@@ -0,0 +1,155 @@
+/*
+ * This file is part of the Sulu CMS.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+define([
+    'services/sulusecurity/role-manager',
+    'services/sulusecurity/role-router'
+], function(RoleManager, RoleRouter) {
+
+    'use strict';
+
+    return {
+
+        collaboration: function() {
+            if (!this.options.id) {
+                return;
+            }
+
+            return {
+                id: this.options.id,
+                type: 'roles'
+            };
+        },
+
+        loadComponentData: function() {
+            return RoleManager.loadOrNew(this.options.id);
+        },
+
+        header: function() {
+            var config =  {
+                tabs: {
+                    url: '/admin/content-navigations?alias=roles',
+                    options: {
+                        data: function() {
+                            return this.data;
+                        }.bind(this)
+                    },
+                    componentOptions: {
+                        values: this.data
+                    }
+                },
+                toolbar: {
+                    buttons: {
+                        save: {
+                            parent: 'saveWithOptions'
+                        }
+                    }
+                }
+            };
+
+            if (!!this.options.id) {
```

Comment:
When seeing this, you could actually also add the security checks as e.g. in https://github.com/sulu/sulu/blob/79df1b3a986493a4b5672e123868187556fa160b/src/Sulu/Bundle/MediaBundle/Resources/public/js/components/collections/edit/main.js#L34